### DISCUSSION TITLE
[feed] Voting on feeds

### DISCUSSION THREAD

**Discussion by sirinath** [fomantic/Fomantic-UI#1453] [Views (card, comment, item, statistic,...)] [answered]
Is it possible to add voting on feeds?

**Comment by lubber-de** [comment]
Sure, just use the [rating module](https://fomantic-ui.com/modules/rating.html) inside the meta group (or wherever you want it)
See https://jsfiddle.net/t07qhdLx/
![ratingfeed](https://user-images.githubusercontent.com/18379884/81066006-ba6ee300-8edc-11ea-9086-583d9fc6850d.gif)

**Comment by sirinath** [comment]
What I had in mind is stack overflow style voting. Maybe you can have a voting module or it can be part of the rating module.

**Comment by lubber-de** [comment, marked as answer]
Fomantic lets you being as creative as you want 😉 
Such voting markup is basically done by some additional html markup. 
See https://jsfiddle.net/gq4sypfa/1/
Take this as a start to whatever you want to implement.

![sovotingfeed](https://user-images.githubusercontent.com/18379884/81091034-eef39680-8efe-11ea-8e7d-3079ab555609.gif)
